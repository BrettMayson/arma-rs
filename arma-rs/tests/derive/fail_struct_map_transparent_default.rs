use arma_rs_proc::FromArma;

#[derive(FromArma)]
#[arma(transparent, default)]
struct Container {
    first: String,
}

#[derive(FromArma)]
#[arma(transparent)]
struct Field {
    #[arma(default)]
    first: String,
}

fn main() {}
