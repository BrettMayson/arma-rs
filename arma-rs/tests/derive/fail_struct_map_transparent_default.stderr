error: #[arma(transparent)] and #[arma(default)] cannot be used together
 --> tests/derive/fail_struct_map_transparent_default.rs:3:10
  |
3 | #[derive(FromArma)]
  |          ^^^^^^^^
  |
  = note: this error originates in the derive macro `FromArma` (in Nightly builds, run with -Z macro-backtrace for more info)

error: #[arma(transparent)] and #[arma(default)] cannot be used together
 --> tests/derive/fail_struct_map_transparent_default.rs:9:10
  |
9 | #[derive(FromArma)]
  |          ^^^^^^^^
  |
  = note: this error originates in the derive macro `FromArma` (in Nightly builds, run with -Z macro-backtrace for more info)
